setup.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8; -*-
+
+import os
+
+from setuptools import setup
+
+
+def read(path):
+    with open(os.path.join(os.path.dirname(__file__), path)) as f:
+        return f.read()
+
+
+long_description = read("README.md")
+
+setup(
+    name="check-synology",
+    version="0.4.0",
+    url="https://github.com/wernerfred/check_synology",
+    author="Frederic Werner",
+    author_email="wernerf@fim.uni-passau.de",
```

Comment:
Please adjust or advise accordingly if you are using a different email address these days.

Reply:
I would rather like to not have my mail address specified anywhere. Is this argument optional? Otherwise we could use my noreply mail address provided by github in the privacy settings

Reply:
I think it can be removed altogether.

Reply:
Done.